packages/ui/src/components/va-tree-view/hooks/useTreeKeyboardNavigation.ts

```diff
@@ -0,0 +1,180 @@
+import { ExtractPropTypes } from 'vue'
+
+import type { TreeNode, TreeView, TreeViewEmitsFunc } from '../types'
+import { useTreeViewProps } from './useTreeHelpers'
+
+type TreeNodeElement = HTMLElement | null
+type UseTreeKeyboardNavigationMethods = Pick<TreeView, 'toggleNode' | 'toggleCheckbox'> & {
+  emit: TreeViewEmitsFunc,
+}
+
+const useTreeKeyboardNavigation = (
+  props: ExtractPropTypes<typeof useTreeViewProps>,
+  methods: UseTreeKeyboardNavigationMethods,
+) => {
+  const { emit, toggleNode, toggleCheckbox } = methods
+  const isElementExpanded = (currentElement: TreeNodeElement): boolean =>
+    currentElement?.getAttribute('aria-expanded') === 'true' || false
```

Comment:
```suggestion
    !!currentElement?.getAttribute('aria-expanded') === 'true'
```

Reply:
Get attribute returns string value.

Reply:
You can still remove "|| false"

Reply:
> Get attribute returns string value.

Yea, just inattentive look. Here just `currentElement?.getAttribute('aria-expanded') === 'true'` will be enough.